hack/generate-manifest-flow-aggregator.sh

```diff
@@ -138,7 +144,12 @@ mkdir configMap && cd configMap
 # but instead to the generated YAML manifest, so our regexs need not be too robust.
 cp $KUSTOMIZATION_DIR/base/conf/flow-aggregator.conf flow-aggregator.conf
 if [[ $FLOW_COLLECTOR != "" ]]; then
-    sed -i.bak -E "s/^[[:space:]]*#[[:space:]]*externalFlowCollectorAddr[[:space:]]*:[[:space:]]\"\"+[[:space:]]*$/externalFlowCollectorAddr:  \"$FLOW_COLLECTOR\"/" flow-aggregator.conf
+    perl -i -p0e 's/  # Enable is the switch to enable exporting flow records to external flow collector.\n  #enable: false/  # Enable is the switch to enable exporting flow records to external flow collector.\n  enable: true/' flow-aggregator.conf
+    sed -i.bak -E "s/^[[:space:]]*#[[:space:]]*address[[:space:]]*:[[:space:]]\"\"+[[:space:]]*$/  address: \"$FLOW_COLLECTOR\"/" flow-aggregator.conf
+fi
+
+if $CLICKHOUSE; then
+    perl -i -p0e 's/  # Enable is the switch to enable exporting flow records to ClickHouse.\n  #enable: false/  # Enable is the switch to enable exporting flow records to ClickHouse.\n  enable: true/' flow-aggregator.conf
```

Comment:
I hope the perl dependency will not be an issue. But I understand that multiline sed is not practical. We can improve later if needed.

Reply:
Got it, thanks!

Reply:
@dreamtalen Can you open a github issue for getting rid of the perl dependency? Just to make sure we will remember to do that before it becomes a problem!